### DISCUSSION TITLE
How to realize devicetree with naming DEVICE_DT_GET(DT_CHOSEN(XXX)) relationship?

### DISCUSSION THREAD

**Discussion by YMont** [zephyrproject-rtos/zephyr#70351] [Q&A]
Hi, All

I have a question that is how to realize devicetree with **zephyr_bt_c2h_uart** relationship?
Where can I define it cause I include it that will be compile error? please more detail. 
How do I activate it if I want add this line code on other main function?
Thanks a lot.

`zephyr\samples\bluetooth\hci_uart\src\main.c`
```c
static const struct device *const hci_uart_dev =
	DEVICE_DT_GET(DT_CHOSEN(zephyr_bt_c2h_uart));
```

**Comment by ndrs-pst** [comment]
@YMont Please take a look at `zephyr\samples\bluetooth\hci_uart\boards`

```
&uart1 {
	current-speed = <1000000>;
	hw-flow-control;
};

/ {
	chosen {
		zephyr,bt-c2h-uart=&uart1;
	};
};
```
